Sources/Controllers/MessagesViewController+Keyboard.swift

```diff
@@ -68,8 +76,11 @@ extension MessagesViewController {
         messageCollectionViewBottomInset = newBottomInset
```

Comment:
We also wanted to add a new property to `MessagesViewController` so users can add additional inset. Something like, `open var additionalBottomInset: CGFloat = 0` and then add this with `newBottomInset`

Reply:
Yeah I think this is a really good idea as a fallback if a user needs to adjust things on their own